src/main/java/pl/bajerska/befindyourmeal/SecurityConfig.java

```diff
@@ -1,22 +1,65 @@
 package pl.bajerska.befindyourmeal;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+import pl.bajerska.befindyourmeal.user.UserDetailsServiceImpl;
+import pl.bajerska.befindyourmeal.user.UserSimpleUrlAuthenticationSuccessHandler;
+
+import javax.sql.DataSource;
 
 @Configuration
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 
+
+    private UserDetailsServiceImpl userDetailsService;
+
+    public SecurityConfig(UserDetailsServiceImpl userDetailsService) {
+        this.userDetailsService = userDetailsService;
+    }
+
+    @Autowired
```

Comment:
Please consider autowiring in the constructor https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/